WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/network/JsonTransferer.java

```diff
@@ -38,10 +39,11 @@ public JsonTransferer() {
 	}
 
 	/**
-	 * Sends the recorded T's to the server. Returns whether or not the transfer
+	 * Sends the recorded items to the server. Returns whether or not the transfer
```

Comment:
I agree with you on the too generic naming. I think WatchDogItem as substitute for WatchDogTransferable is a good candidate solution, especially because WatchDogTransferables do not necessarily have to be transferred.